docs/UserGuide.md

```diff
@@ -151,7 +152,9 @@ Format: `editp INDEX (must be a positive integer) [n/NAME] [p/PHONE] [e/EMAIL] [
     - have each domain label consist of alphanumeric characters, separated only by hyphens, if any.
 * The length of the `ADDRESS` must be between 6 and 70 characters
 * The length of the `REMARK` must be less than or equal to 70 characters
-* The length of a tag must be less than 70 characters.
+* The length of a tag must be less than or equal to 70 characters.
+  * If the tag prefix is left empty E.g. `t/`, it will remove all tags that are currently associated to the person
+  * If tags are is provided, any existing tags will be removed and replaced by the tags provided
```

Comment:
slight grammatical error here, other than that LGTM

Reply:
Fixed! Thanks!